src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -152,6 +153,62 @@ static DataTestUtility()
             }
         }
 
+        public static string SetupFileStreamDB()
+        {
+            try
+            {
+                if (!IsFileStreamDbSetup())
+                {
+                    if (!FileStreamDirectory.EndsWith("\\"))
+                    {
+                        FileStreamDirectory += "\\\\";
```

Comment:
Why the double `\`s on the end of the path?
```suggestion
                        FileStreamDirectory += "\\";
```

Reply:
I guess I escaped twice